MarvelApiClient/DefaultHeadersRequestInterceptor.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  DefaultHeadersRequestInterceptor.swift
+//  MarvelAPIClient
+//
+//  Created by Pedro Vicente Gomez on 04/01/16.
+//  Copyright © 2016 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamNetworking
+
+class DefaultHeadersRequestInterceptor: BothamRequestInterceptor {
```

Comment:
IMHO the interceptor mechanism is quite cool :) As you say, we can provide this implementation by default as part of BothamNetworking.